### PR TITLE
YOTTANK glorification

### PR DESCRIPTION
**Pull request by Runakai1**
Added remainining 4 cores + cells to YOTTANK + modified recipe cost, was unreasonably high.
Made the Multiblock allow up to cosmic neutronium borosilicate glas

Closes: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10476

New recipes:
T3 Core:
image
T4 Core:
image
T5 Core:
image
T6 Core:
image
T7 Core:
image
T8 Core:
image
T9 Core:
image
T10 Core:
image

Cell Recipes:
T6:
image
T7:
image
T8:
image
T9:
image
T10:
image

### CODE REVIEW

**Review by Connor-Colenso — DISMISSED** [review]
I would use the tier appropriate parts rather than doubling up (these go to UMV now). Also super coolant rather than coolant at these tiers ideally.

**Comment by Runakai1** [comment]
> I would use the tier appropriate parts rather than doubling up (these go to UMV now). Also super coolant rather than coolant at these tiers ideally.

Done

File: src/main/java/goodgenerator/loader/RecipeLoader_02.java [hunk 1]
```diff
@@ -566,10 +736,34 @@ public static void RecipeLoad() {
 
         GT_Values.RA.addUnboxingRecipe(
                 ItemRefer.YOTTank_Cell_T6.get(1),
-                ItemRefer.Fluid_Storage_Core_T6.get(10),
+                ItemRefer.Fluid_Storage_Core_T6.get(2),
```
Comment (GlodBlock) [inline src/main/java/goodgenerator/loader/RecipeLoader_02.java:739]: the recursive recipe should be kept so people can recycle and upgrade the existed tanks

**Review by GlodBlock — DISMISSED** [review]

**Comment by Dream-Master** [comment]
@GlodBlock

**Comment by GlodBlock** [comment]
uh, the spotless check failed

**Comment by Dream-Master** [comment]
Yea

**Comment by Dream-Master** [comment]
I mean beside this the pr ok or we make the game to easy @GlodBlock @Connor-Colenso ?

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**